front/src/components/Artwork.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="artwork">
+    <div class="top">
+      <a class="title info">{{ artwork.name }}</a>
```

Comment:
Why not use `id=` instead of `class=` ?

Reply:
Same for 
- artwork
- top
- description
- image
- author
- location
- ...